commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResultBase.java

```diff
@@ -6,12 +6,14 @@
 
 public abstract class PagedResultBase<T> extends Base implements PagedResult<T> {
     protected final Long offset;
+    protected final Long limit;
     protected final Long total;
     protected final List<T> results;
     protected final Long count;
 
-    protected PagedResultBase(final Long offset, final Long total, final List<T> results, final Long count) {
+    protected PagedResultBase(final Long offset, final Long limit, final Long total, final List<T> results, final Long count) {
```

Comment:
So I have added everywhere the limit next to the offset because they belong together, but on the other side it might be error-prone for people extending this class as it's a list of Longs and limit is in the middle. You mistake the position and you got it wrong. WDYT?

Reply:
I think that it's ok to have this somehow ugly constructor. The only way we could improve it would require wrapping the longs into a separate parameter class , but then the constructor of this parameter class would have the same ugliness 😜  And it's the responsibility of the user of this class to correctly call this constructor in a sub class and to write a test for it.